src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int defaultCircleRadius = 10;
+    private final String defaultCircleColor = "white";
```

Comment:
Please use constant naming convention for 'defaultCircleColor' by making it uppercase and static final: 'DEFAULT_CIRCLE_COLOR'.
and get color from Enum
```suggestion
    private final String DEFAULT_CIRCLE_COLOR = Color.WHITE.name();
```